docs/user-guide/researcher/model-testing-during-federated-training.md

```diff
@@ -149,8 +153,25 @@ exp.set_set_test_metric(MetricTypes.ACCURACY)
 exp.run(rounds=2, increase=True)
 ```
 
+## Using Validation facility with heavy datasets
 
+In some specific cases, you may have some very huge datain your datasets, such as 3D images, or you want a huge validation set, that you can load on `Node`s only through batches, in order to avoid `Nodes` to crash due to lack of Memory. `Fed-BioMed` provides an option to compute validation metrics through batches, in the `TrainingArgument`: `test_batch_size`.
```

Comment:
```suggestion
Depending on data type, using whole dataset can use too much space in memory (e.g 3D images) or some datasets can have high number of observations that consumes too much memory. If whole dataset is used during the testing nodes can crash due to lack of memory or computation takes longer. To avoid such issues `Fed-BioMed` provides an option to compute validation metrics through batches, in the `TrainingArgument`: `test_batch_size`.
```